notify/impl.go

```diff
@@ -220,19 +220,21 @@ func NewEmail(c *config.EmailConfig, t *template.Template, l log.Logger) *Email
 // auth resolves a string of authentication mechanisms.
 func (n *Email) auth(mechs string) (smtp.Auth, error) {
 	username := n.conf.AuthUsername
-
+	err := &types.MultiError{}
 	for _, mech := range strings.Split(mechs, " ") {
 		switch mech {
 		case "CRAM-MD5":
 			secret := string(n.conf.AuthSecret)
 			if secret == "" {
+				err.Add(errors.New("Missing secret for CRAM-MD5 auth mechanism"))
```

Comment:
I think errors should start with a lower case letter: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
You are right, I updated errors and unit tests.